src/main/kotlin/nl/tudelft/ewi/se/ciselab/testgenie/services/TestCaseDisplayService.kt

```diff
@@ -111,6 +111,7 @@ class TestCaseDisplayService(private val project: Project) {
         selectAllButton.addActionListener { toggleAllCheckboxes(true) }
         deselectAllButton.addActionListener { toggleAllCheckboxes(false) }
         toggleJacocoButton.addActionListener { toggleJacocoCoverage() }
+        removeAllButton.addActionListener { removeAllTestCases() }
```

Comment:
Didn't i add that before?

Reply:
Yes, but someone broke it during a merge.